src/styles/header.css

```diff
@@ -1,42 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
 * {
   list-style-type: none;
   text-decoration: none;
 }
 
+.image-li {
+  width: 50%;
+  text-align: end;
+}
+
 header {
   width: 100%;
+  margin: auto;
   border-bottom: 0.1px solid grey;
-  background-color: white;
   margin-bottom: 25px;
 }
 
 nav {
   width: 90%;
-  height: 100px;
-  display: flex;
   margin: auto;
-  align-items: center;
+  height: 5.5rem;
+  padding: 1.438rem 6.188rem 1.688rem 2.8rem;
+  background-color: var(--white-two);
 }
```

Comment:
- ***Your Design***
![image](https://github.com/ClintonYounge/My-Bookstore-Project/assets/60083437/3ff20c5c-3768-442d-bce0-37f7c69290e2)

- ***Given Design***
![image](https://github.com/ClintonYounge/My-Bookstore-Project/assets/60083437/a74a505e-cb14-41ee-ae54-f346d64a0eeb)

- [ ] Color of the navigation menu such as: `Books` and `Categories` are not matching with the given design. From the given design you can see when a menu is `clicked` it becomes `darker` such as `Books`, and when a  menu is `unclicked` it becomes `lighter` such as `Categoies`. Kindly follow the design guideline for the color and use it in your design